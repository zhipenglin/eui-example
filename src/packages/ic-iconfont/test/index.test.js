import React from 'react';
import IcIcon from '../src/index';
import renderer from 'react-test-renderer';

jest.mock('../src/fonts/multicolor/iconfont',()=>'iconfont');

it('renders correctly', () => {
    const tree = renderer.create(<IcIcon className="ic-tag-icon" type="close_small" size={14}></IcIcon>).toJSON();
    expect(tree).toMatchSnapshot();
});
