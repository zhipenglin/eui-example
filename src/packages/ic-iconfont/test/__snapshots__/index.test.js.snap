// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<i
  className="ic-icon icon-close_small ic-tag-icon"
  style={
    Object {
      "fontSize": "14px",
    }
  }
/>
`;
